readme.txt

```diff
@@ -0,0 +1,67 @@
+=== Komoju Payments ===
+Contributors: degica
+Tags: WooCommerce,Payment Gateway,Komoju
+Requires at least: 5.3
+Tested up to: 5.4
+Stable tag: trunk
+Requires PHP: 7.2
+License: MIT
+License URI: https://directory.fsf.org/wiki/License:X11
+
+A WooCommerce plugin to receive payments with Komoju, Japan's leading provider for online and offline payments.
+
+== Description ==
+
+The Komoju Payments WooCommerce plugin allows customers to checkout with a variety of Japanese payment options, include Credit Card, Konbini, Webmoney and more.
+
+To use this plugin you must have a [Komoju account](https://komoju.com/).
+
+= Configuration Instructions =
+Login to Wordpress as Administrator 
+
+Click on the 'Plugins' menu on the left hand side.
+
+You should see the WooCommerce Komoju Gateway listed among the plugins list. Click the Activate Link for this gateway.
```

Comment:
Think we should remove the word list from the end of the first sentence

Reply:
Done in 743ecbf